dissect/target/tools/info.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import json
+import logging
+
+from dissect.target import Target
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.tools.query import record_output
+from dissect.target.tools.utils import (
+    configure_generic_arguments,
+    process_generic_arguments,
+)
+
+InfoRecord = TargetRecordDescriptor(
+    "target/info",
+    [
+        ("datetime", "last_activity"),
+        ("datetime", "install_date"),
+        ("net.ipaddress[]", "ips"),
+        ("string", "os_family"),
+        ("string", "os_version"),
+        ("string", "architecture"),
+        ("string", "language"),
+        ("string", "timezone"),
+    ],
+)
+
+
+log = logging.getLogger(__name__)
+logging.lastResort = None
+logging.raiseExceptions = False
+
+
+def main():
+    """target-info"""
+
+    help_formatter = argparse.ArgumentDefaultsHelpFormatter
+    parser = argparse.ArgumentParser(
+        description="dissect.info",
+        fromfile_prefix_chars="@",
+        formatter_class=help_formatter,
+        add_help=False,
+    )
+    parser.add_argument("targets", metavar="TARGETS", nargs="*", help="Targets to display info from")
+    parser.add_argument("--from-file", nargs="?", help="file containing targets to load")
+    parser.add_argument("-d", "--delimiter", default=" ", action="store", metavar="','")
+    parser.add_argument("-s", "--strings", action="store_true", help="print output as string")
+    parser.add_argument("-r", "--record", action="store_true", help="print output as record")
+    parser.add_argument("-j", "--json-pretty", action="store_true", help="output records as pretty json")
+    parser.add_argument("-J", "--json", action="store_true", help="output records as one-line json")
+    configure_generic_arguments(parser)
+
+    args, rest = parser.parse_known_args()
+
+    if not args.targets and ("-h" in rest or "--help" in rest):
+        parser.print_help()
+        parser.exit()
```

Comment:
```suggestion
    args = parser.parse_args()
```
This tool doesn't have to do anything with additional arguments, so can just use `parse_args()` here.